torntools/scripts/content/stockexchange/ttStockexchange.js

```diff
@@ -486,12 +486,43 @@ function getDiffClass(diff) {
 
 function showTotalPortfolioValue() {
 	let totalPortfolioValue = 0;
-	for (let element of document.findAll(".stock-main-wrap>ul.stock-cont>li.item-wrap")) {
+	for (let element of document.findAll(
+		"div.stock-main-wrap>ul.stock-cont>li.item-wrap"
+	)) {
 		totalPortfolioValue += parseInt(
-			element.find(".item .info .c-price-wrap .first-row span").innerText.replace("Worth: ", "").replace("$", "").replace(/,/g, "").trim()
+			element
+				.find(".item .info .c-price-wrap .first-row span")
+				.innerText.replace("Worth: ", "")
+				.replace(/[$,]/g, "")
+				.trim()
 		);
 	}
-	totalPortfolioValue = numberWithCommas(totalPortfolioValue);
-	totalPortfolioValue = "$" + totalPortfolioValue.toString();
-	document.find(".title").innerText += ` (Total Value ${totalPortfolioValue})`;
+	totalPortfolioValue =
+		"$" + numberWithCommas(totalPortfolioValue).toString();
+
+	let profitsSum = (lossSum = rawText = null);
+	doc.findAll(".bold.block-profit").forEach(
```

Comment:
I wouldn't use a `forEach` for this, but rather a `reduce`.

`const profitsSum = [...doc.findAll(".bold.block-profit")].map((x) => parseInt(x.innerText.replace(/[$\+, ]/g, "")).reduce((a, b) => a += b, 0)`